python/llm/test/benchmark/arc-perf-test.yaml

```diff
@@ -27,3 +27,6 @@ in_out_pairs:
 test_api:
   - "transformer_int4_gpu"  # on Intel GPU
 cpu_embedding: False # whether put embedding to CPU (only avaiable now for gpu win related test_api)
+exclude:
+  - 'fnlp/moss-moon-003-sft:1024'
+  - 'fnlp/moss-moon-003-sft:2048'
```

Comment:
Need to add `'bigscience/bloomz-7b1:2048'` which causes OOM error and segment fault randomly.